presto-docs/src/main/sphinx/release/release-0.193.rst

```diff
@@ -0,0 +1,71 @@
+=============
+Release 0.193
+=============
+
+General Changes
+---------------
+
+* Fix an infinite loop during planning for queries containing non-trivial predicates.
+* Fix a bug that causes queries that constrain the result of ``row_number()`` to numbers less than or
+  equal to 0 to fail or return incorrect results.
+* Fix failure during query planning when lambda expressions are used in ``UNNEST`` or ``VALUES`` clauses.
+* Fix ``Tried to free more revocable memory than is reserved`` error for queries that run in the
+  reserved memory pool with spilling enabled.
+* Improve the performance of the :func:`ST_Contains` function.
+* Add :func:`map_zip_with` lambda function.
+* Add :func:`normal_cdf` function.
+* Add ``SET_DIGEST`` type and related functions.
+* Add query stat to keep track of peak total memory.
+* Improve performance of queries that filter all data from a table up-front (e.g., due to partition pruning).
+* Turn on new local scheduling algorithm by default (see :doc:`release-0.181`)
+* Remove the ``information_schema.__internal_partitions__`` table.
+
+Security Changes
+----------------
+
+* Apply the authentication methods in the order they are listed in the
+  ``http-server.authentication.type`` configuration.
+
+CLI Changes
+-----------
+
+* Fix rendering of maps of Bing tiles.
+* Abort the query correctly when the result pager exits.
+
+JDBC Driver Changes
+-------------------
+
+* Use SSL by default for port 443.
+
+Hive Changes
+------------
+
+* Fix a bug that causes a ``ConcurrentModificationException`` when dropping any column
```

Comment:
```
* Allow dropping any column in a table. Previously, dropping columns other
  than the last one would fail with ``ConcurrentModificationException``.